src/embed_tests/pyrunstring.cs

```diff
@@ -24,10 +24,10 @@ public void Dispose()
         public void TestRunSimpleString()
         {
             int aa = PythonEngine.RunSimpleString("import sys");
-            Assert.AreEqual(aa, 0);
```

Comment:
@yagweb I think you had these backwards by mistake, its the opposite other than python. I fixed them before merge.